packages/core/src/react/blocks/YoutubeLiteBlock.tsx

```diff
@@ -28,6 +28,7 @@ declare global {
 export interface YoutubeLiteBlockProps extends IBlockAttributes {
 	src: string;
 	title: string;
+	params: string;
```

Comment:
This should be an optional parameters

```suggestion
	params?: string;
```